Library/ViewModels/NoShippingPostCampaignCheckoutViewModel.swift

```diff
@@ -238,7 +238,8 @@ public class NoShippingPostCampaignCheckoutViewModel: NoShippingPostCampaignChec
         project: data.project,
         total: pledgeTotal,
         confirmationLabelHidden: true,
-        pledgeHasNoReward: pledgeHasNoRewards(rewards: rewardsData.rewards)
+        pledgeHasNoReward: pledgeHasNoRewards(rewards: rewardsData.rewards),
+        pledgeOverTimeData: nil
```

Comment:
Is pledge over time an option for late pledges? If it is, why is this hardcoded to nil? If it's not, why are the `PostCampaignPledgeRewardsSummaryTotalViewController` changes required?

Reply:
In this version (v1), we won’t support late pledges, but we will in V2. For now, it’s hardcoded to `nil` because we don’t need it here at the moment.

Regarding your last question, the changes in `PostCampaignPledgeRewardsSummaryTotalViewController` are required because this controller is used on the checkout screen to display the `Pledge Amount`, just below the `Your Pledge` table.

<img width="320" alt="Screenshot 2024-12-13 at 11 36 17 a m" src="https://github.com/user-attachments/assets/bdb6f7db-bde4-47a5-8890-e5d325167230" />

![image](https://github.com/user-attachments/assets/5f11d6c2-630e-4d01-ad17-6c9a912ab9b3)